### ISSUE TITLE
Add filling of time.Time

### ISSUE THREAD

**Issue by chrisaxiom** [axiomzen/golorem#3]
Currently missing this functionality

**Comment by chrisaxiom** [comment]
Seems not possible: https://play.golang.org/p/GmFedGO1wr

**Comment by bradleymcallister97** [comment]
@chrisaxiom could you not set the value like this https://play.golang.org/p/_8ziuqPP4J7

**Comment by chrisaxiom** [comment]
@bradleymcallister97 yes I believe this is what I have done in later code (have to set it a level up and not treat it as any old struct)

**Comment by bradleymcallister97** [comment]
I think it would be great to also add the ability to fill *apd.Decimal values
